docs/user/authentication_and_authorization.md

```diff
@@ -131,13 +131,16 @@ There are several possible answers:
                 }
             } 
         ], 
-        "rCanWrite": [ 
+        "canWrite": [ 
             … 
         ], 
-        "rCanAdmin": [ 
+        "canAdmin": [ 
             … 
-        ], 
-        "@context": [ "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/authorization/jsonld-contexts/authorization.jsonld" ] 
+        ],
+        "isOwner": {
+            // only if user is not the owner of the entity, in which case the value of the "right" property will be "isOwner"
```

Comment:
you mean only if user **is** the owner of the entity ?

Reply:
no, I mean "is not". if it is the owner, the information will be in the `right` attribute (and not duplicated here).